sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+
+        private java.lang.Object rulesUpdatedAt_ = "";
+        /**
+         * <code>optional string rulesUpdatedAt = 11;</code>
+         */
+        public boolean hasRulesUpdatedAt() {
+          return ((bitField0_ & 0x00000400) == 0x00000400);
+        }
+        /**
+         * <code>optional string rulesUpdatedAt = 11;</code>
+         */
+        public java.lang.String getRulesUpdatedAt() {
+          java.lang.Object ref = rulesUpdatedAt_;
+          if (!(ref instanceof java.lang.String)) {
+            com.google.protobuf.ByteString bs =
+                (com.google.protobuf.ByteString) ref;
+            java.lang.String s = bs.toStringUtf8();
+            if (bs.isValidUtf8()) {
+              rulesUpdatedAt_ = s;
+            }
+            return s;
+          } else {
+            return (java.lang.String) ref;
+          }
+        }
+        /**
+         * <code>optional string rulesUpdatedAt = 11;</code>
+         */
+        public com.google.protobuf.ByteString
+            getRulesUpdatedAtBytes() {
+          java.lang.Object ref = rulesUpdatedAt_;
+          if (ref instanceof String) {
+            com.google.protobuf.ByteString b = 
+                com.google.protobuf.ByteString.copyFromUtf8(
+                    (java.lang.String) ref);
+            rulesUpdatedAt_ = b;
+            return b;
+          } else {
+            return (com.google.protobuf.ByteString) ref;
+          }
+        }
+        /**
+         * <code>optional string rulesUpdatedAt = 11;</code>
+         */
+        public Builder setRulesUpdatedAt(
+            java.lang.String value) {
+          if (value == null) {
+    throw new NullPointerException();
+  }
+  bitField0_ |= 0x00000400;
+          rulesUpdatedAt_ = value;
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional string rulesUpdatedAt = 11;</code>
+         */
+        public Builder clearRulesUpdatedAt() {
+          bitField0_ = (bitField0_ & ~0x00000400);
+          rulesUpdatedAt_ = getDefaultInstance().getRulesUpdatedAt();
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional string rulesUpdatedAt = 11;</code>
+         */
+        public Builder setRulesUpdatedAtBytes(
+            com.google.protobuf.ByteString value) {
+          if (value == null) {
+    throw new NullPointerException();
+  }
+  bitField0_ |= 0x00000400;
+          rulesUpdatedAt_ = value;
+          onChanged();
+          return this;
+        }
+
+        // @@protoc_insertion_point(builder_scope:sonarqube.ws.rules.WsSearchResponse.QualityProfile)
+      }
+
+      static {
+        defaultInstance = new QualityProfile(true);
+        defaultInstance.initFields();
+      }
+
+      // @@protoc_insertion_point(class_scope:sonarqube.ws.rules.WsSearchResponse.QualityProfile)
+    }
+
+    public static final int PROFILES_FIELD_NUMBER = 1;
+    private java.util.List<org.sonarqube.ws.QualityProfiles.WsSearchRespo
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 242 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "profilesBuilder_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)